src/client/hud.cpp

```diff
@@ -322,7 +322,10 @@ void Hud::drawLuaElements(const v3s16 &camera_offset)
 					NULL, colors, true);
 				break; }
 			case HUD_ELEM_TEXT: {
-				irr::gui::IGUIFont* textfont = (e->size.X > 0) ? g_fontengine->getFont(e->size.X * g_fontengine->getDefaultFontSize()) : font;
+				irr::gui::IGUIFont* textfont = font;
+				if (e->size.X > 0)
+					textfont = g_fontengine->getFont(e->size.X * g_fontengine->getDefaultFontSize());
```

Comment:
It still seems like this line is pretty long (102 chars, with a tab width of 4) - maybe split this line after `getFont(`?

```suggestion
					textfont = g_fontengine->getFont(
						e->size.X * g_fontengine->getDefaultFontSize());
```

Reply:
There's a line below a ways that's 101 characters long, so I'm not sure

Reply:
Eh, why not